### PR TITLE
Bounty Requests: Move to django forms

### PR DESCRIPTION
**Pull request by pinkiebell**
@SaptakS @mbeacom 
Feedback on this one? 😃

### CODE REVIEW

File: app/bounty_requests/templates/bounty_request_form.html [hunk 1]
```diff
@@ -38,46 +38,36 @@ <h3>{% trans "Request a Bounty" %} <span id="alpha">{% trans "Alpha" %}</span></
...
+                    {% if not user.is_authenticated %}
+                    (<a href="{% url 'social:begin' 'github' %}?next={{ request.get_full_path }}">{% trans 'Login' %}</a>)
+                    {% endif %}
+                  </label>
+                  <input name="requested_by" placeholder="{% trans "Please Login First" %}" id="requested_by" class="form__input" type="text" value="{% if github_handle %}{{ github_handle }}{% endif %}" required disabled />
+                </div>
+                <form id="bounty_request_form">
+                  {% for field in form %}
+                    <div class="mb-3">
+                      <label for="{{ field.name }}" class="form__label">{{ field.label }}</label>
```
Comment (SaptakS) [inline app/bounty_requests/templates/bounty_request_form.html]: The `for` attribute binds it to the id of the element and not the name. So for this to work, I think you need to set the id for the elements same as the name in the form widget.

Comment (pinkiebell) [reply app/bounty_requests/templates/bounty_request_form.html]: @SaptakS Whoa. Yes you are right, seems like I'm getting too old 😆 .

Comment (SaptakS) [reply app/bounty_requests/templates/bounty_request_form.html]: 😅

File: app/bounty_requests/templates/bounty_request_form.html [hunk 2]
```diff
@@ -49,7 +49,7 @@ <h3>{% trans "Request a Bounty" %} <span id="alpha">{% trans "Alpha" %}</span></
                 <form id="bounty_request_form">
                   {% for field in form %}
                     <div class="mb-3">
-                      <label for="{{ field.name }}" class="form__label">{{ field.label }}</label>
+                      <label for="id_{{ field.name }}" class="form__label">{{ field.label }}</label>
```
Comment (SaptakS) [inline app/bounty_requests/templates/bounty_request_form.html]: 🙇

**Comment by mbeacom** [comment]
:shipit: 🚢

**Pull request merged**